frontend/app/components/authentication/signup.hbs

```diff
@@ -1,47 +1,65 @@
-          </BsAlert>
+        <BsAlert @type="danger" @fade={{true}}>
+          {{err.validation}}
+        </BsAlert>
         {{/if}}
-      {{/each}}
+        {{/each}}
+
+        <form.element id="password" class="" @controlType="password" @label={{t "auth.login.password" }}
+          @property="password" as |el|>
+          <el.control autocomplete="new-password" placeholder={{t "auth.login.enter_password" }}></el.control>
+        </form.element>
+
+        <form.element id="confirm-password" class="" @controlType="password" @label={{t "auth.signup.repeat_password" }}
+          @property="passwordConfirmation" as |el|>
+          <el.control placeholder={{t "auth.signup.repeat_password" }} autocomplete="new-password"></el.control>
+        </form.element>
+
+        <form.element id="invite-code" class="" @controlType="text" @label={{t "auth.signup.invite_code" }}
+          @property="inviteCode" as |el|>
+          <el.control placeholder={{t "auth.signup.invite_code" }}></el.control>
+        </form.element>
+      </div>
 
-      <form.element id="password" class="" @controlType="password" @label={{t "auth.login.password"}} @property="password" as |el|>
-        <el.control autocomplete="new-password" placeholder={{t "auth.login.enter_password"}}></el.control>
-      </form.element>
+      <BsButton class="btn btn-primary form-control mb-2" @type="submit" @defaultText={{t "auth.signup.signup" }}
+        @buttonType="submit" />
 
-      <form.element id="confirm-password" class="" @controlType="password" @label={{t "auth.signup.repeat_password"}}
-                    @property="passwordConfirmation" as |el|>
-        <el.control placeholder={{t "auth.signup.repeat_password"}}  autocomplete="new-password"></el.control>
-      </form.element>
+      <BsButton type="button" class="bg-primary form-control mb-2" {{ on "click"
+        this.authenticateWithGoogleImplicitGrant}}>
+        <span class="px-2">{{svg-jar "google-icon"}}</span> {{t "auth.signup.signup_third_party" authProvider="Google"}}
+      </BsButton>
 
-      <form.element id="invite-code" class="" @controlType="text" @label={{t "auth.signup.invite_code"}}  @property="inviteCode" as |el|>
-        <el.control placeholder={{t "auth.signup.invite_code"}}></el.control>
-      </form.element>
-    </div>
+      <BsButton type="button" class="bg-primary form-control mb-2" {{ on "click" this.authenticateWithFacebook}}>
+        <span class="px-2">{{svg-jar "facebook-icon"}}</span>{{t "auth.signup.signup_third_party"
+        authProvider="Facebook"}}
+      </BsButton>
 
-    <BsButton class="btn btn-primary form-control mb-2" @type="submit" @defaultText={{t "auth.signup.signup"}} @buttonType="submit"/>
+      <BsButton type="button" class="bg-primary form-control mb-2" {{ on "click" this.authenticateWithGitHub}}>
+        <span class="px-2">{{svg-jar "github-icon"}}</span>{{t "auth.signup.signup_third_party" authProvider="GitHub"}}
+      </BsButton>
```

Comment:
the third party auth buttons on the login page looks awesome :1st_place_medal:  

Can we kindly have the same consistency of button designs across the signup & teach login pages?.

Reply:
Thats a WIP, still working with Yego to standardize the designs -- too many buttons and the page look bloated.